lib/matplotlib/axes/_axes.py

```diff
@@ -1240,7 +1245,44 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
 
         return colls
 
+    def _plot_args_replacer(self, args, data):
```

Comment:
This needs to be not a instance method as it gets bound in without the self and raises execptions when you use `plot`.